src/views/lighttable.c

```diff
@@ -1184,6 +1184,8 @@ void gui_init(dt_view_t *self)
   gtk_overlay_add_overlay(GTK_OVERLAY(dt_ui_center_base(darktable.gui->ui)), lib->preview->widget);
   gtk_overlay_reorder_overlay(GTK_OVERLAY(dt_ui_center_base(darktable.gui->ui)),
                               gtk_widget_get_parent(dt_ui_log_msg(darktable.gui->ui)), -1);
+  gtk_overlay_reorder_overlay(GTK_OVERLAY(dt_ui_center_base(darktable.gui->ui)),
```

Comment:
Can we really have toast message on the lighttable ?

Same question for the map below ?

Reply:
Not right now but if someone decides to add dynamic accels to lib module sliders or comboboxes... Up to you I can remove it if you like.